### PR TITLE
Added CROSS guidance

### PR DESCRIPTION
**Pull request by markwilkie**
Before turning on CROSS, we need to make sure we have the right guidance in place so contributors can understand what's happening.

Please note, CROSS only "reports" when there's something found, and historically speaking, more than 90% of the PRs are fine.

### CODE REVIEW

File: Documentation/project-docs/contributing.md [hunk 1]
```diff
@@ -28,6 +28,11 @@ Maintainers will not merge changes that have narrowly-defined benefits, due to c
 
 Contributions must also satisfy the other published guidelines defined in this document.
 
+Automated Code Review Assistance
+------------------
+
+CROSS is a tool developed my Microsoft that we use to highlight areas of higher risk in a code change in order to help code reviewers do a more effective job
```
Comment (Petermarcu) [inline Documentation/project-docs/contributing.md]: my -> by.

"we use" -> "is used"

End it with a period. :)

**Comment by markwilkie** [comment]
Updated

**Comment by Petermarcu** [comment]
LGTM. Looks like CROSS check succeeded :)

**Comment by richlander** [comment]
The text is good but it doesn't say much about how contributors should react to CROSS messages or what those messages will look like. How will I know that a message is a CROSS message, for example?

**Comment by terrajobst** [comment]
> The text is good but it doesn't say much about how contributors should react to CROSS messages

That's a good point. Maybe we should say add something like this:

> We deeply care about security issues in our code and we thoroughly review each PR. However, humans aren’t perfect and will make mistakes. To aid our code reviews, we run a tool called [CROSS](cross.md) that inspects the PR to find interesting areas that need further inspection. If the tool finds such areas, it will add a comment to the PR letting everyone know about this. If you’re interested to learn more, take a look at [CROSS](cross.md). **However, in general contributors aren't expected to take any actions. The project maintainers will use the comments posted by [CROSS](cross.md) to inform their PR feedback.**

(addition is bolded)

**Comment by Petermarcu** [comment]
@terrajobst , I think you are operating on an old diff. @richlander I think the text is pretty self explanatory for contributors. Can't anyone look at the results as an assist for reviewing the code?

**Comment by terrajobst** [comment]
Yes, I took the older commit but I only did to provide context for the addition.

I think @richlander has point: contributors aren't necessarily aware how to read the report. Do they have to change their code? Is it a false positive? I think explicitly stating that the maintainers are meant to take action removes that confusion. However, this doesn't disallow feedback from contributors either :-)

**Pull request merged**